src/lotto/main.py

```
@@ -1,6 +1,140 @@
+import random
```

Ruff (0.8.2)

`random` imported but unused

Remove unused import: `random`

(F401)